examples/flutter_gallery/lib/gallery/home.dart

```diff
@@ -154,7 +154,7 @@ class _CategoriesPage extends StatelessWidget {
                       );
                     }),
                   );
-                }),
+                })..insert(0,               CircularProgressIndicator()),
```

Comment:
With this hack, you will see that the gallery app continually renders frames when displaying the pesto page because the main page is active.